client/extensions/woocommerce/app/promotions/promotion-update.js

```diff
@@ -191,8 +207,10 @@ function mapStateToProps( state, ownProps ) {
 	const promotion = promotionId ? getPromotionWithLocalEdits( state, promotionId, site.ID ) : null;
 	const products = getPromotionableProducts( state, site.ID );
 	const productCategories = getProductCategories( state, site.ID );
+	const hasEdits = Boolean( getPromotionEdits( state, promotionId, site.ID ) );
 
 	return {
+		hasEdits,
```

Comment:
Likewise on the `propType` addition here.

Reply:
👍